workers_kv_test.go

```diff
@@ -0,0 +1,369 @@
+package cloudflare
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWorkersKV_CreateStorageNamespace(t *testing.T) {
+	setup(UsingOrganization("foo"))
+	defer teardown()
+
+	response := `{
+		"result": {
+			"id" : "3aeaxxxxee014exxxx4cf66xxxxc0448",
+			"title": "test_namespace"
+		},
+		"success": true,
+		"errors": [],
+		"messages": []
+	}`
+
+	mux.HandleFunc("/accounts/foo/storage/kv/namespaces", func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "POST", r.Method, "Expected method 'POST', got %s", r.Method)
+		w.Header().Set("content-type", "application/javascript")
+		fmt.Fprintf(w, response)
+	})
+
+	res, err := client.CreateStorageNamespace(context.Background(), &StorageNamespaceRequest{Title: "Namespace"})
+	want := StorageNamespaceResponse{
+		successResponse,
+		"3aeaxxxxee014exxxx4cf66xxxxc0448",
+		"test_namespace",
+	}
+
+	if assert.NoError(t, err) {
+		assert.Equal(t, want.Response, res.Response)
+	}
```

Comment:
```suggestion
	require.NoError(t, err)
	require.Equal(t, want.Response, res.Response)
```

Reply:
I don't see a require dependency installed in this repository. Are you asking me to install something?

Reply:
Ahh it's part of testify. I understand now. Updating.